### PR TITLE
fixed the broken link and update readme file

### PR DESCRIPTION
**Pull request by jsroyal**
Hi! @karussell Please review it. Is this broken link you talking about it?
#338 
Also, fixed the syntax in getting started readme file.

### CODE REVIEW

**Comment by jsroyal** [comment]
It would be great to merge this PR because one of them I 'm using frequently and sometimes see the page is not available and after that, I have to go manually

**Comment by jsroyal** [comment]
@karussell @oblonski, @braktar @pierredavidbelanger
Please review and merge it.

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ jsprit
 [![Build Status](https://travis-ci.org/graphhopper/jsprit.svg?branch=master)](https://travis-ci.org/graphhopper/jsprit)
 
 jsprit is a java based, open source toolkit for solving rich <a href="http://en.wikipedia.org/wiki/Travelling_salesman_problem" target="_blank">traveling salesman</a> (TSP) and <a href="http://neo.lcc.uma.es/vrp/vehicle-routing-problem/" target="_blank">vehicle routing problems</a> (VRP).
-It is lightweight, flexible and easy-to-use, and based on a single all-purpose <a href="https://github.com/jsprit/jsprit/wiki/Meta-Heuristic" target="_blank">meta-heuristic</a> currently solving
+It is lightweight, flexible and easy-to-use, and based on a single all-purpose <a href="https://github.com/jsroyal/jsprit/blob/master/docs/Meta-Heuristic.md" target="_blank">meta-heuristic</a> currently solving
```
Comment (braktar) [inline README.md]: Hi !
Seems to have a bad link here ;)
jsroyal -> graphhopper

Comment (jsroyal) [inline README.md]: Sorry for that mistake.
@braktar Thank you for review.

**Comment by jsroyal** [comment]
I think it would be good in one PR.

**Pull request closed without merging**